config/admins.txt

```diff
@@ -12,3 +12,4 @@ duncandelano = Game Master
 ecolli = Game Master
 crazypilot = Game Master
 vivalas = Coder
+Qustinnus = Game Master
```

Comment:
you realize when you're testing it locally it automatically makes you an admin, right?

if it doesn't, you aren't testing your code correctly